### PR TITLE
feat: Add tracker host in Dashboard

### PR DESCRIPTION
**Pull request by flashlab**
#  Add tracker host in Dashboard

This will be useful when we use qBit as PT client. Especially when we have many reseeded enties across seveal sites.
more details are in #676 .

# PR Checklist

- [x] I've started from master
- [x] I've only committed changes related to this PR
- [x] All Unit tests pass
- [x] I've removed all commented code
- [x] I've removed all unneeded console.log statements

### CODE REVIEW

File: src/components/Torrent/DashboardItems/Tracker.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export default defineComponent({
+  name: 'Tracker',
+  mixins: [TorrentDashboardItem],
+  props: {
+    torrent: Torrent
+  },
+  computed: {
+    trackerString() {
+      let trackerUrl = this.torrent.tracker.match(/:\/\/([^\/]+\.)?([^\/\.]+)\.[^\/\.:]+/i)
```
Comment (WDaan) [inline src/components/Torrent/DashboardItems/Tracker.vue:25]: Can we extract this matching, that's also used in the other function to a helper or something?

Perhaps write some unit tests for it to guarantee its functionality. 🤔

**Review by WDaan — APPROVED** [review]
Thanks for becoming a contributor! Cool feature :) 

Left some small remark that would improve maintainability.

File: src/components/Torrent/DashboardItems/Tracker.vue (hunk 1 above)
Comment (flashlab) [reply src/components/Torrent/DashboardItems/Tracker.vue]: I'm not familiar with nodejs, maybe the second commit will be better ;)

**Comment by WDaan** [comment]
Awesome! 😎

**Pull request merged**